File: Modeling/app/src/main/java/pisarev/com/modeling/mvp/model/Point.java
package pisarev.com.modeling.mvp.model;

public class Point {

    protected float x = Constant.N_GANTRYPOS_X;

    protected float z = Constant.N_GANTRYPOS_Z;

    public Point() {

    }

    public Point(Point point){
        this.x= point.getX();
        this.z= point.getZ();
    }

    public Point(float x, float z) {
        this.x = x;
        this.z = z;
    }

    public float getX() {
        return x;
    }

    public void setX(float x) {
        this.x = x;
    }

    public float getZ() {
        return z;
    }

    public void setZ(float z) {
        this.z = z;
    }
}

File: Modeling/app/src/main/java/pisarev/com/modeling/mvp/contur/Point.kt
package pisarev.com.modeling.mvp.contur

import pisarev.com.modeling.mvp.model.core.GCode

open class Point : GCode {

    var x = N_GANTRYPOS_X.toFloat()
    var z = N_GANTRYPOS_Z.toFloat()

    constructor() {}

    constructor(point: Point) {
        this.x = point.x
        this.z = point.z
    }
}